res/css/views/rooms/_EventTile.pcss

```diff
@@ -648,6 +648,32 @@ $left-gutter: 64px;
             margin-top: 0;
```

Comment:
(I don't really know why we need a `margin-top` here. As far as I can tell, it's the default anyway)

Reply:
Me nether.